index.html

```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My conference capstone project</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.3/css/all.css" 
+    integrity="sha384-SZXxX4whJ79/gErwcOYf+zWLeJdY/qpuqC4cAa9rOGUstPomtqpuNWT9wdPEn2fk" 
+    crossorigin="anonymous">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700&display=swap">
+</head>
+<body>
+    <header class="header">
+        <nav class="nav">
+            <div class="icon" id="bars">
+                <i class="fas fa-bars"></i>
+            </div>
+            <div class="logo">
+                <a  href="index.html">
+                   <h2>CGDMC.2021</h2>  
+                </a>
+            </div>
+            <ul class="top-nav">
+                <li class="top-nav__item">
+                    <a href="#" class="top-nav__link">
+                        <i class="fab fa-twitter"></i>
+                    </a>
+                </li>
+                <li class="top-nav__item">
+                    <a href="#" class="top-nav__link">
```

Comment:
- [ ] On the mobile screen there is a white border existing around the screen, kindly remove it and ensure the page content fills the entire screen.

| Original | Yours |
| ----- | ------ |
| <img src='https://user-images.githubusercontent.com/33323077/128993673-b2152025-e91b-44a4-a8f4-8a527c89ea23.png' height='250px'> | <img src='https://user-images.githubusercontent.com/33323077/128993741-99d597cf-40cd-4083-bd38-f4684cf49fc2.png' height='250px'> |